.github/workflows/php.yml

```diff
@@ -119,7 +119,7 @@ jobs:
         run: composer install --ansi --prefer-dist --no-interaction --no-progress
 
       - name: Run phpunit (without DB)
-        run: ./vendor/phpunit/phpunit/phpunit -c tests/Unit/phpunit.xml
+        run: ./vendor/phpunit/phpunit/phpunit -c tests/Unit/phpunit.xml; git diff --exit-code
```

Comment:
```suggestion
        run: ./vendor/phpunit/phpunit/phpunit -c tests/Unit/phpunit.xml
```

Reply:
No question ? No argue ? ^^' ...
I didn't make this change like a random bot xD

Then I changed this cause phpunit command changed some git versioned files and this behavior wasn't detected by the CI but induced some mess in local environment after running tests.

Do you have any suggestion to ensure that tests won't break anything in environment ? :slightly_smiling_face:

Reply:
You can add a final test in Github Action where to check that the output is empty.

No chain commands, please.

Reply:
Thanks for explanations !! ^^